fvtest/tril/tril/ast.hpp

```diff
@@ -76,13 +101,17 @@ struct ASTValue {
      * is_FloatintPoint_Compatible<double>::value; // true
      * is_String_Compatible<const char*>::value;   // true
      */
+#if __cplusplus <= 199711L
+     // Never used for the version without C++11 support.
```

Comment:
I'd rather flip the sense of this check, and avoid the 

```
#if FOO 
//nothing
#else
stuff
#endif
```
pattern. It can be replaced with 

```
#if !FOO
stuff
#endif
```